js/scripts.js

```diff
@@ -12,21 +12,22 @@ function createTweet(input) {
 	var quoteAuthor = data.title;
 	var tweetText = "Quote of the day - " + quoteText + " Author: " + quoteAuthor;
 
-	if(!quoteAuthor.length) {
-		quoteAuthor = "Unknow author";
-	}
-	
-	if (tweetText.lenght > 140) {
-		getQuote();
-	} else {
-		var tweet = tweetLink + encodeURIComponent(tweetText);
-		$('.quote').text(quoteText);
-		$('.author').text("Author: " + quoteAuthor);
-		$('.tweet').attr('href', tweet);
+	if (input.length > 0) {
```

Comment:
too many ifs ;) use negation to prepare a smarter solution with the same result but simpler code ;)